README.md

```diff
@@ -1 +1,23 @@
-# hw01
\ No newline at end of file
+# Bio of Kat Edmiston
+
+## A bit about me...
+My Name is Kat Edmiston, and I'm a rising fourth year. I'm getting a [joint BA/MA with the MAPSS program!](https://mapss.uchicago.edu/) I'm majoring in [Law, Letters, and Society](http://collegecatalog.uchicago.edu/thecollege/lawlettersandsociety/) and at least one of me theses will look at the relationship between loosening charter school regulations and school performance through non-traditional metrics (i.e. not testing).
+
+My interests include:
+* Baking
+  + (Primarily cookies and pastries, though I'm attempting croissants this week.)
+* Research into alternative education metrics
+* Yoga
+* Watching dumb competition shows, like:
+  + [Forged in Fire](https://www.history.com/shows/forged-in-fire) ![Forged in Fire](https://static.next-episode.net/tv-shows-images/huge/forged-in-fire.jpg)
+  + [Chopped](https://www.foodnetwork.com/shows/chopped) ![Chopped](https://www.seriouseats.com/images/2012/08/20120808-chopped.jpg)
+  + [Blown Away](https://en.wikipedia.org/wiki/Blown_Away_(TV_series)) ![Blown Away](https://blog.cmog.org/wp-content/uploads/2019/07/BlownAway_1x1_POST.jpg)
+  + and [Project Runway](https://www.bravotv.com/project-runway) ![and Project Runway](https://www.thewrap.com/wp-content/uploads/2020/01/Project-Runway.jpg)
+***
```

Comment:
Also, I saw in your comments that you were having trouble with making captions for the pictures. If you were to fix that this would be great!